src/Apps/Components/AppContainer.tsx

```diff
@@ -0,0 +1,8 @@
+import { breakpoints } from "@artsy/palette"
+import styled from "styled-components"
+
+export const AppContainer = styled.div`
+  width: 100%;
+  max-width: ${breakpoints.xl}px;
```

Comment:
Takes the largest possible breakpoint from our [palette Theme file](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L18) and applies it here.